configuration/configuring-zed.md

```diff
@@ -181,19 +182,34 @@ To override settings for a language, add an entry for that languages name to the
 
 For a complete list of languages you can override settings for see [Supported Languages](../languages/support-languages.md).
 
+## Font Features
+
+Zed supports a subset of OpenType features that can be enabled or disabled for a given buffer or terminal font. For example, to disable ligatures for a given font you can add the following to your settings:
+
+```json
+{
+  "buffer_font_features": {
+    "calt": false
+  }
+}
+```
+
+Optionally, the following OpenType features can be enabled or disabled too: `calt`, `case`, `cpsp`, `frac`, `liga`, `onum`, `ordn`, `pnum`, `ss01`, `ss02`, `ss03`, `ss04`, `ss05`, `ss06`, `ss07`, `ss08`, `ss09`, `ss10`, `ss11`, `ss12`, `ss13`, `ss14`, `ss15`, `ss16`, `ss17`, `ss18`, `ss19`, `ss20`, `subs`, `sups`, `swsh`, `titl`, `tnum`, `zero`.
```

Comment:
Looks great! 😄. Would it make sense to also link out to a canonical site that explains what each option means? That's if such a site exists. Maybe this one? https://en.wikipedia.org/wiki/List_of_typographic_features

```suggestion
Optionally, the following [OpenType features](https://en.wikipedia.org/wiki/List_of_typographic_features) can be enabled or disabled too: `calt`, `case`, `cpsp`, `frac`, `liga`, `onum`, `ordn`, `pnum`, `ss01`, `ss02`, `ss03`, `ss04`, `ss05`, `ss06`, `ss07`, `ss08`, `ss09`, `ss10`, `ss11`, `ss12`, `ss13`, `ss14`, `ss15`, `ss16`, `ss17`, `ss18`, `ss19`, `ss20`, `subs`, `sups`, `swsh`, `titl`, `tnum`, `zero`.
```